src/renderer/components/dock/__test__/log-tab.store.test.ts

```diff
@@ -1,152 +0,0 @@
-/**
```

Comment:
I think these tests that I removed quite useless atm and buggy, most important tests should be introduced for base class in  `__tests__/dock-tab.store.ts` (on my TODO list for this PR).

Regarding to `log-tab.store.test.ts` it might be revived later in separated PR with extra additional tests for every class inherited from `DockTabStore`. How does it sound?

Reply:
That sounds good enough. Though moving to `Singleton` should help with testing